src/chatgptClient.ts

```diff
@@ -23,6 +23,16 @@ export async function getSuggestions(
 
   // extract the json from the response
   const result = response.data.choices[0].text ?? ''
+
+  // eslint-disable-next-line no-console
```

Comment:
### Line 27
## CodeGuard Suggestions
**Suggestion:** Use 'lastIndexOf' instead of indexOf to get the exact last index of '}'
**Reason:** It is easier to match and get the exact index of the wanted character with this method